packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/VideoResultTask.kt

```diff
@@ -25,7 +25,7 @@ class VideoResultTask(private val promise: Promise,
       val outputFile = fileProvider.generateFile()
       saveVideo(outputFile)
       val response = Bundle().apply {
-        putString("uri", outputFile.toURI().toString())
+        putString("uri", outputFile.toURL().toString())
```

Comment:
This method is deprecated 😞 
https://docs.oracle.com/javase/7/docs/api/java/io/File.html#toURL()
What about using the suggestion from the deprecation message? 🤔 
```suggestion
        putString("uri", outputFile.toURI().toURL().toString())
```

Reply:
`outputFile.toURI().toURL()` also returns incorrect path. 
I believe that Android changed how URIs are parsed and now we should use "file:/" instead of "file:///". But we can't migrate right now. We rely on "file:///" in many places. So, I've added a method that will replace "file:/" with "file:///".

Reply:
Ok, so you're saying that:
- Android changed it's internal behaviour (it would be nice to have some reference)
- we are using some other approach that the default one

I'm ok with the change, but we need to investigate this and unify across all codebase, ideally to follow native Android behaviour 😉 I'll create an issue for that.